container-images/rocm-fedora-gfx10/Containerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
Rather then have 4 Containerfiles, that we need to manage can we just consolidate this into one ROCM containerfile and then manage this with container_build.sh and build_llama_whisper.sh

These containerfiles have very little difference that I can see then which files are being removed for size.

If users say make `build IMAGE=rocm-fedora-gfx10` Then this should be enough to change the `rm -fr ...` line and potentially the FROM line.

Reply:
@rhatdan the problem is the resulting image is like 40GB , do we want people to have to pull a container image of that size?

There's little getting around it other than splitting.

The reason the UBI9 one doesn't suffer from this is it only includes support for a small subset of AMD GPUs

Reply:
We could just include the UBI9 subset in the Fedora image and add other individual GPUs on demand (like @maxamillion 's GPU). That's another option.

Reply:
I see what you mean not @rhatdan I'd just add another argument to the build_llama_and_whisper.sh script if it was me, would remove these lines:

```
RUN dnf update -y --refresh && dnf clean all
...
ENV WHISPER_CPP_SHA=${WHISPER_CPP_SHA}
ENV LLAMA_CPP_SHA=${LLAMA_CPP_SHA}
```